lib/v2/datasource/remote/model/proposals_model.dart

```diff
@@ -0,0 +1,88 @@
+enum ProposalType { alliance, campaign, hypha }
+
+class ProposalsModel {
+  final List<ProposalModel> proposals;
+
+  ProposalsModel(this.proposals);
+
+  factory ProposalsModel.fromJson(Map<String, dynamic> json) {
+    if (json['rows'] != null) {
+      return ProposalsModel(json['rows'].map((i) => ProposalModel.fromJson(i)).toList());
+    } else {
+      return ProposalsModel([]);
+    }
+  }
+}
+
+class ProposalModel {
+  final int? id;
+  final String? creator;
+  final String? recipient;
+  final String? quantity;
+  final String? staked;
+  final int? executed;
+  final int? total;
+  final int? favour;
+  final int? against;
+  final String? title;
+  final String? summary;
+  final String? description;
+  final String? image;
+  final String? url;
+  final String? status;
+  final String? stage;
+  final String? fund;
+  final int? creationDate;
+
+  ProposalType get type {
+    return fund == 'allies.seeds'
```

Comment:
a simple if else if else its easier to follow than this long ternary operation .

Reply:
I actually copied it for reference because I still don't know where it's used or if it's even necessary. If not, I will remove it in the following PRs.